backend/src/main/java/com/ody/mate/service/MateService.java

```diff
@@ -43,14 +50,32 @@ public MateSaveResponseV2 saveAndSendNotifications(
             Member member,
             Meeting meeting
     ) {
+        validateMeetingOverdue(meeting);
         validateAlreadyAttended(member, meeting);
+        Mate mate = saveMateAndEta(mateSaveRequest, member, meeting);
+
+        FcmTopic fcmTopic = new FcmTopic(meeting);
+        sendEntry(meeting, mate, fcmTopic);
+        notificationService.subscribeTopic(member.getDeviceToken(), fcmTopic);
+        sendDepartureReminder(meeting, mate, fcmTopic);
```

Comment:
[제안]
notificationService 관련 로직은 하나의 private 메서드로 분리하면 어떤가요?
```suggestion
        sendEntryAndDepartureReminder()
```
sendEntry(), sendDepartureReminder()는 private 메서드인데, subscribeTopic만 바로 호출되고 있어서, 차라리 하나로 묶이면 더 가독성이 좋을 것 같아요.

Reply:
당시 이렇게 구현했다고 전해들었던 이유는 입장알림은 입장을 한 당사자가 받지 않게하기 위함으로 알고 있어요.

입장알림을 보내고 > 구독하고 > 리마인더를 보내야 참여하는 mate를 제외한 약속 참여원들이 입장알림을 받고, 당사자가 받지 않게 되는 것으로 알고 있습니다.

Reply:
순서를 변경하자는 의미는 아니었어요.
sendEntry(), subscribeTopic(), sendDepartureReminder()를 sendEntryAndDepartureReminder() 혹은  sendEntryAndSubscribeAndDepartureReminder()..? private 메서드로 분리하면 가독성이 더 좋겠다라는 의미입니다😸